internal/packages/assets.go

```diff
@@ -0,0 +1,171 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package packages
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/elastic-package/internal/multierror"
+)
+
+// AssetType represents the type of package asset.
+type AssetType string
+
+// Supported asset types.
+const (
+	AssetTypeElasticsearchIndexTemplate  AssetType = "index_template"
+	AssetTypeElasticsearchIngestPipeline AssetType = "ingest_pipeline"
+
+	AssetTypeKibanaSavedSearch   AssetType = "search"
+	AssetTypeKibanaVisualization AssetType = "visualization"
+	AssetTypeKibanaDashboard     AssetType = "dashboard"
+	AssetTypeKibanaMap           AssetType = "map"
```

Comment:
+ Lens

Reply:
Addressed in 133e50ee0364bb76f289086a0ca4aa16b92c6967.